streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -58,6 +62,13 @@
     @Deprecated
     public final InternalTopologyBuilder internalTopologyBuilder = new InternalTopologyBuilder();
 
+    private Topology.AutoOffsetReset translateAutoOffsetReset(final TopologyBuilder.AutoOffsetReset resetPolicy) {
```

Comment:
I see. Ack, on removing all var/method annotations. Will do a final cleanup in fourth PR.